persistence/rest/src/main/java/org/infinispan/persistence/rest/configuration/RestStoreConfigurationParser.java

```diff
@@ -142,12 +142,8 @@ private void parseRestStoreAttributes(XMLExtendedStreamReader reader, RestStoreC
          String attributeName = reader.getAttributeLocalName(i);
          Attribute attribute = Attribute.forName(attributeName);
          switch (attribute) {
-            case APPEND_CACHE_NAME_TO_PATH: {
-               builder.appendCacheNameToPath(Boolean.parseBoolean(value));
-               break;
-            }
-            case PATH: {
-               builder.path(value);
+            case CACHE_NAME: {
+               builder.cacheName(value);
```

Comment:
I would rather keep parsing the old attributes with older schemas, like you do in the subsystem reader, and only throw an exception if we can't infer a cache name.

Reply:
sounds good, I will use the same strategy